pkg/dynamicinstrumentation/codegen/c/dynamicinstrumentation.c

```diff
@@ -64,12 +64,12 @@ int {{.GetBPFFuncName}}(struct pt_regs *ctx)
     event->base.uid = uid;
 
     // Collect stack trace
-    __u64 currentPC = ctx->pc;
+    __u64 currentPC = ctx->__PT_IP_REG;
```

Comment:
I think the preferred way is this: 
```suggestion
    __u64 currentPC = PT_REGS_IP(ctx);
```